src/tests/suite/daos_container.c

```diff
@@ -1881,6 +1881,157 @@ co_owner_implicit_access(void **state)
 	test_teardown((void **)&arg);
 }
 
+static void
+expect_co_set_attr_access(test_arg_t *arg, uint64_t perms, int exp_result)
+{
+	daos_prop_t	*cont_prop;
+	int		 rc = 0;
+	const char	*name = "AttrName";
+	const char	*value = "This is the value";
+	const size_t	 size = strnlen(value, 32);
+
+	cont_prop = get_daos_prop_with_owner_acl_perms(perms,
+						       DAOS_PROP_CO_ACL);
+
+	while (!rc && arg->setup_state != SETUP_CONT_CONNECT)
```

Comment:
does arg->cont_open_flags need to be assigned in this function? Or maybe default is RW?

Reply:
Default is indeed RW.